lib-src/rpc2/include/rpc2/pack_helper.h

```diff
@@ -0,0 +1,106 @@
+/* BLURB lgpl
+
+                           Coda File System
+                              Release 5
+
+          Copyright (c) 1987-2016 Carnegie Mellon University
+                  Additional copyrights listed below
+
+This  code  is  distributed "AS IS" without warranty of any kind under
+the  terms of the  GNU  Library General Public Licence  Version 2,  as
+shown in the file LICENSE. The technical and financial contributors to
+Coda are listed in the file CREDITS.
+
+                        Additional copyrights
+
+#*/
+
+/*
+                         IBM COPYRIGHT NOTICE
```

Comment:
I don't think this contains any code from 1986, so the IBM copyright notice should not apply